### PR TITLE
Add a func to check the WLCG path

### PR DESCRIPTION
**Pull request by biozit**
A function to check if a request is WLCG.

### CODE REVIEW

File: Collectors/wlcg_converter.py [hunk 1]
```diff
@@ -183,6 +184,22 @@ def Convert(source_record):
     
     return to_return
 
+# check the path on TPC     
+def tpcPathCheckWLCG(pathCheck):
+    try:
+        parts =  re.split('(.*):\/\/([\w\-\.]+)(\:*[0-9]*)(\/+)(.*)',pathCheck)
+    
+        path = parts[5]
```
Comment (ddavila0) [inline Collectors/wlcg_converter.py:192]: Just send this line (192) to after the "if" to prevent and out of index error when the "if" statement is true

**Review by ddavila0 — CHANGES_REQUESTED** [review]
ignore this

**Comment by ddavila0** [comment]
LGTM

**Comment by biozit** [comment]
@djw8605 @jthiltges  could you check this?

**Comment by jthiltges** [comment]
Did the urllib.parse alternative work out? URL parsing by regex is painful, and it would be nice to avoid it if we can.

```python
import urllib.parse

def tpcPathCheckWLCG(path):
    parts = urllib.parse.urlparse(path)
    path = parts.path.lstrip('/')
    if path.startswith('store') or path.startswith('user/dteam'):
        return True
    else:
        return False
```

**Comment by biozit** [comment]
The urllib.parse is 11 times faster than the regex!!

**Comment by biozit** [comment]
@jthiltges how about the code now?

File: Collectors/wlcg_converter.py [hunk 2]
```diff
@@ -35,6 +35,8 @@
 import uuid
 import time
 import socket
+import re
```
Comment (jthiltges) [inline Collectors/wlcg_converter.py:38]:
```suggestion
```

**Review by jthiltges — CHANGES_REQUESTED** [review]
LGTM aside from the `re` import that is unused

**Review by jthiltges — APPROVED** [review]
LGTM

**Pull request merged**